grape-pipeline.nf

```diff
@@ -578,26 +539,27 @@ process markdup {
     template(command)
 }
 
-if ( params.markDuplicates || params.removeDuplicates ) {
-    bamFilesToGenome = markdupOutput
-}
 
-bamFilesToGenome.into {
-    inferExpInput
-    bamStatsInput
-}
+
```

Comment:
I would clean up commented old code like this

Reply:
Done